torchprime/experimental/torchax_models/deepseek_v3/tests/test_prefill.py

```diff
@@ -1,23 +1,35 @@
-import pytest
+import unittest
 
+import torch
+import torchax
+import torchax.interop
+from torchprime.experimental.torchax_models.deepseek import model as ds_model
 
-# TODO(https://github.com/AI-Hypercomputer/torchprime/issues/75): Fix the failure on torch 2.6,
-# then enable the test unconditionally.
-@pytest.mark.deepseek
-def test_single_device_compile():
-  from torchprime.experimental.torchax_models.deepseek_v3.prefill_benchmark import (
-    single_device_compile,
-  )
 
-  single_device_compile()
+class DeepseekModuleTest(unittest.TestCase):
 
+  def setUp(self):
+    torchax.enable_globally()
 
-# TODO(https://github.com/AI-Hypercomputer/torchprime/issues/75): Fix the failure on torch 2.6,
```

Comment:
Did the torch 2.6 compat problem got fixed? This was meant to exclude the test from 2.6 runs and only run it with 2.4.

Reply:
hmm good point, let;s use the old way to be sure